regulations/static/regulations/js/source/views/comment-review-view.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var CommentView = require('./comment-view');
+
+var CommentReviewView = Backbone.View.extend({
+  events: {
+    'click #submit': 'submit'
+  },
+
+  initialize: function(options) {
+    Backbone.View.prototype.setElement.call(this, '#' + options.id);
+    this.template = $('#comment-template').html();
+    this.$content = this.$el.find('#comment');
+    this.docNumber = this.$el.data('doc-number');
+    this.prefix = 'comment:' + this.docNumber;
+
+    this.showComments();
+  },
+
+  render: function() {},
+
+  showComments: function() {
+    this.comments = _.chain(_.keys(window.localStorage))
+      .filter(function(key) {
+        return key.indexOf(this.prefix) === 0;
```

Comment:
Would it make more sense to namespace rather than stepping through everything in localStorage? E.g.
`window.localStorage.commentEditor[this.docNumber].comments`